__tests__/app.test.js

```diff
@@ -65,23 +65,78 @@ describe("app", () => {
                 expect(typeof res.body.topics).toBe("object")
                 expect(res.body.topics).toHaveLength(3)
             })
+        });
+    })
+
+    describe.only("GET /api/articles/:article_id/comments", () => {
+        test("should return an array of objects with expected properties", () => {
+            return request(app).get('/api/articles/1/comments').expect(200).then((res) => {
+                const result = res.body.comments
 
```

Comment:
The use of for in here is a little strange and may throw some folks off. For in can be used on arrays, but is much more commonly used with objects. I would personally use either a forEach() or a for loop here